create-module.sh

```diff
@@ -228,7 +230,7 @@ packageJson="$cssPath/package.json"
 echo -e "Creating ${CYAN}$packageJson${NC}"
 cat > $packageJson << EOF
 {
-	"name": "@workday/canvas-kit-css-$name",
+	"name": "@workday/canvas-kit-css-$moduleName",
```

Comment:
These module names are still incorrect

Reply:
I've got local changes that will address these issues, lmk if we want it committed on this branch.

Reply:
#211 addresses these issues.